crud/example_test.go

```diff
@@ -85,6 +85,81 @@ func ExampleResult_rowsCustomType() {
 	// [{{} 2010 45 bla}]
 }
 
+// ExampleTuples_customType demonstrates how to use a slice of objects of a
+// custom type as Tuples to make a ReplaceManyRequest.
+func ExampleTuples_customType() {
+	conn := exampleConnect()
+
+	// The type will be encoded/decoded as an array.
+	type Tuple struct {
+		_msgpack struct{} `msgpack:",asArray"` //nolint: structcheck,unused
+		Id       uint64
+		BucketId *uint64
+		Name     string
+	}
+	req := crud.MakeReplaceManyRequest(exampleSpace).Tuples([]Tuple{
+		Tuple{
+			Id:       2010,
+			BucketId: nil,
+			Name:     "bla",
+		},
+	})
+
+	ret := crud.MakeResult(reflect.TypeOf(Tuple{}))
+	if err := conn.Do(req).GetTyped(&ret); err != nil {
+		fmt.Printf("Failed to execute request: %s", err)
+		return
+	}
+
+	fmt.Println(ret.Metadata)
+	rows := ret.Rows.([]Tuple)
+	fmt.Println(rows[0].Id)
+	fmt.Println(*rows[0].BucketId)
+	fmt.Println(rows[0].Name)
```

Comment:
Is this guaranteed to have at least one element? Maybe it will be more correct to check the `rows` size so we don't get a panic on example?

Reply:
Updated.